src/App.js

```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Header from './components/Header';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
     </div>
   );
```

Comment:
I noticed that you may have forgotten to import the InputTodo component into your App.js file. This could be why you're not seeing the InputTodo component rendered on the page.

To fix this issue, make sure that you import the InputTodo component in your App.js file using the appropriate import statement. For example:

`import React from 'react';
import InputTodo from './InputTodo';
// other imports

function App() {
  // your code here
  return (
    <div className="App">
      <InputTodo />
      // other components
    </div>
  );
}
export default App;
`